Cargo.toml

```diff
@@ -16,6 +16,7 @@ managed = { version = "0.5", default-features = false, features = ["map"] }
 byteorder = { version = "1.0", default-features = false }
 log = { version = "0.3", default-features = false, optional = true }
 libc = { version = "0.2.18", optional = true }
+bitflags = { version = "1.0", default-features = false, optional = true }
```

Comment:
I would prefer if we had `bitflags` non-optional for simplicity, since it has zero runtime cost.

Reply:
:+1: